apps/ftu/js/wifi.js

```diff
@@ -230,8 +231,10 @@ var WifiUI = {
     // Retrieve SSID from dataset
     var ssid = event.target.dataset.ssid;
 
+    var selectedNetwork = WifiManager.getNetwork(ssid);
```

Comment:
I'd rather made it promise-like now than filling a followup, if that's ok with you.